### DISCUSSION TITLE
Purpose of this counter?

### DISCUSSION THREAD

**Discussion by raka-gunarto** [iPortalTeam/ImmersivePortalsMod#1616] [General]
https://github.com/iPortalTeam/ImmersivePortalsMod/blob/1.21/src/main/java/qouteall/imm_ptl/core/portal/nether_portal/BreakablePortalEntity.java#L247-L256

I'm jut having a read of the internals to get a better understanding of how the mod works and using it in my own mod development. I came across this odd code here. What is the purpose of this counter? This always returns true which removes it from the tasklist on the first run effectively making it a one-shot. Should this be <= instead? And why 30 times?

**Comment by qouteall** [comment]
Oh yes you found a coding bug. I will fix it.

**Comment by qouteall** [comment]
Thanks.

**Comment by qouteall** [comment]
It used to be to handle the case where the other side chunk loading is delayed.
